neurom/analysis/morphtree.py

```diff
@@ -44,8 +44,13 @@
 
 def path_length(tree):
```

Comment:
Btw, we have to be careful with the handling of the memory as well because when I did the loading with the 1000 synthesized cells from Guy (I think that they had a relatively small number of segments), they took up 13 gb of ram. Maybe it would be a good approach to do memoization with limited capacity?